index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav-bar">
+            <a class="logo decoration" href="#">My Logo</a>
+            <ul class="nav-items">
+                <li class="list-style"><a class="decoration" href="#">Portfolio</a></li>
+                <li class="list-style"><a class="decoration" href="#">About</a></li>
+                <li class="list-style"><a class="decoration" href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+        <section class="text-section">
+            <h1>I’m Ibrahim <br/> Glad to see you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
```

Comment:
- [ ] As already suggested by the previous reviewer, please ensure that you follow the design guidelines(fonts, margins, padding, colors etc)
- [ ] Also bear in mind that you are only required to implement the mobile version of the design in this milestone. The desktop design will be implemented in upcoming milestones

### Your Designs
![image](https://user-images.githubusercontent.com/43995830/158821702-3567d657-42ff-4ad3-a582-31db527a7b5c.png)

![image](https://user-images.githubusercontent.com/43995830/158821810-2795e09f-67fb-4d42-a7d1-43c606ba234a.png)

### Original Desgin
![image](https://user-images.githubusercontent.com/70269507/158690374-c36a8ffd-16da-444c-85ac-08cb613790fe.png)